src/Magnum/GL/TransformFeedback.cpp

```diff
@@ -30,6 +30,7 @@
 #ifndef MAGNUM_TARGET_WEBGL
```

Comment:
The `<tuple>` include could be wrapped in an `#ifdef MAGNUM_BUILD_DEPRECATED `.

Reply:
Done